itou/www/apply/forms.py

```diff
@@ -926,6 +926,34 @@ def get_sender_companies_choices(self):
         return sorted(sender_orgs, key=lambda org: org[0])
 
 
+class JobApplicationCommentForCompany(forms.ModelForm):
+    """
+    Allows members of a company to edit a shared common comment on the job application detail page.
+    There are two modes:
+    - edit: the comment can be edited via a textarea
+    - view: the comment can be read via a custom div widget
+    """
+
+    shared_comment = forms.CharField(
+        strip=True,
+        label="Note partagée avec tous les membres de la structure",
+        required=False,
+    )
+
+    class Meta:
+        model = JobApplication
+        fields = [
+            "shared_comment",
+        ]
+
+    def __init__(self, *args, mode="view", **kwargs):
+        super().__init__(*args, **kwargs)
+        if mode == "edit":
+            self.fields["shared_comment"].widget = forms.Textarea(attrs={"data-it-expandable": "true"})
```

Comment:
```suggestion
            self.fields["shared_comment"].widget = forms.Textarea(attrs={"data-it-expandable": "true", "autofocus": "true"})
```

Pour directement avoir le focus dans le textarea (HTMX gère bien ça)